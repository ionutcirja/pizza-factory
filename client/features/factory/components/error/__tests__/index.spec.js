import React from 'react';
import { shallow } from 'enzyme';
import Error from '..';

describe('Error component', () => {
  const propsToRender = {
    message: 'error',
  };
  
  describe('render', () => {
    it('should render an error message', () => {
      const wrapper = shallow(<Error {...propsToRender} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
