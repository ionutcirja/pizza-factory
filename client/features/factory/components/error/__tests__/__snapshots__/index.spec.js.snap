// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component render should render an error message 1`] = `
<span>
  error
</span>
`;
